roles/wordpress-setup/tasks/self-signed-certificate.yml

```diff
@@ -1,10 +1,19 @@
 ---
 - name: Generate self-signed certificates
-  shell: >
-    openssl req -subj "/CN={{ item.value.site_hosts[0].canonical }}" -new
-    -newkey rsa:2048 -days 3650 -nodes -x509 -sha256
-    -keyout {{ item.key }}.key -out {{ item.key }}.cert
+  shell: "openssl req -subj \"/CN={{ item.value.site_hosts[0].canonical }}\" -new \
+    -newkey rsa:2048 -days 3650 -nodes -x509 -sha256 \
+    -extensions req_ext -config <( \
+cat <<\" EOF\"\n
```

Comment:
Ansible issues ansible/ansible#12856 and ansible/ansible#8512 prevent the use of `EOF` or `-EOF` as file delimiter, since a space character is inserted by ansible at the start of each line while parsing the shell string.
If found the solution [here in the second issue](https://github.com/ansible/ansible/issues/8512#issuecomment-102382070).

I don't know if you have a better understanding of the problem than me, but i couldn't find a better solution here or elsewhere.